styleguide/_themes/global/scss/forms.scss

```diff
@@ -37,12 +37,12 @@ table {
   .help-block,
   .radio,
   .radio-inline {
-    color: #eb0022;
+    color: $brand-danger;
   }
 
   .form-control {
     background-color: rgba(235, 0, 34, .1);
```

Comment:
I appreciate the eye for detail, but if the errors are already accounted for we can come back to fix them up in a PR specifically for that. We added the warnings threshold to avoid hold ups like this in code review.